admin/server/auth/handlers.go

```diff
@@ -261,45 +308,103 @@ func (a *Authenticator) authLoginCallback(w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	customDomainFlow, ok := sess.Values[cookieFieldCustomDomainFlow].(bool)
+	delete(sess.Values, cookieFieldRedirect)
```

Comment:
I think this should be:
```suggestion
	delete(sess.Values, cookieFieldCustomDomainFlow)
```